SDWebImageFLPlugin/Classes/FLAnimatedImageBridge/UIImage+SDWebImageFLPlugin.m

```diff
@@ -19,4 +20,18 @@ - (void)setSd_FLAnimatedImage:(FLAnimatedImage *)sd_FLAnimatedImage {
     objc_setAssociatedObject(self, @selector(sd_FLAnimatedImage), sd_FLAnimatedImage, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
 }
 
++ (instancetype)sd_imageWithFLAnimatedImage:(FLAnimatedImage *)animatedImage {
+    UIImage *posterImage = animatedImage.posterImage;
+    CGImageRef imageRef = posterImage.CGImage;
+    if (!imageRef) {
+        return nil;
+    }
+    UIImage *image = [[UIImage alloc] initWithCGImage:imageRef scale:posterImage.scale orientation:UIImageOrientationUp];
```

Comment:
Needs to change `UIImageOrientationUp` to `posterImage. imageOrientation`?

Reply:
GIF does not support EXIF orientation